code/osx32_syscalls.cpp

```diff
@@ -40,4 +46,23 @@ extern "C" {
         return (int)syscall3(SYS_mprotect, (void*)addr, (void*)length,
                              (void*)prot);
     }
+
+    void* memset(void *addr, int val, size_t length) {
+        unsigned char *d = (unsigned char*)addr;
+
+        for (; length; length--, d++)
```

Comment:
I'd prefer to have braces on these loops.

Reply:
Can you make memset match the style in my memcpy implementation?

Reply:
.

Reply:
Updated these functions in 9eb6ebd